src/azure-cli/azure/cli/command_modules/backup/_validators.py

```diff
@@ -42,3 +43,23 @@ def validate_crr(target_rg_id, rehydration_priority):
 
     if rehydration_priority is not None:
         raise MutuallyExclusiveArgumentError("Archive restore isn't supported for secondary region.")
+
+
+def validate_czr(backup_config_response, recovery_point):
+    backup_storage_redundancy = backup_config_response.properties.storage_type
```

Comment:
May I ask will backup_config_response.properties always have property storage_type or cross_region_restore_flag ?

Reply:
Yes. These are compulsory parameters in the response. Attaching the response class for reference. 

<img width="668" alt="image" src="https://user-images.githubusercontent.com/26112239/144013745-112c7086-92ee-4a54-bb44-a9bf77888d54.png">